### ISSUE TITLE
Icons missing roles

### ISSUE THREAD

**Issue by LJWatson** [w3c/webex#12]
The icons and messages displayed in the "Getting started dialog have the following problems:
* The icons are not identified as such to screen reader users;
* The icons and text for the "Help" and "Meeting information" are not presented clearly to screen reader users.

To resolve the first issue:
* Set `role="img"` on the ```<span>``` elements representing the icons;
* Use `aria-label` to provide an alternative text for each icon;
* Hide the text inside the ```<span>``` elements using `display:none;` (so it only comes into use when someone disables CSS and/or uses their own style sheets).

For example:
```
<div class="pb-mod-fte-helperMenu">
<span `role="img" aria-label="Audio"` class="icon-ic50 icon-audio">Audio icon</span>
<span `role="img" aria-label="Participants"` class="icon-ic50 icon-plist">Participant list icon</span>
<span `role="img" aria-label="Chat"` class="icon-ic50 icon-chat">Chat icon</span>
<span `role="img" aria-label="More..."` class="icon-ic50 icon-more">More icon</span>
<span `role="img" aria-label="Leave meeting"` class="icon-ic50 icon-leave">Leave meeting icon</span>
</div>
```

This will cause screen readers to treat each icon like a standard graphic. Note: the display:none; on the inner text can be set in the external style sheet.

To resolve the second of the issues:
* Use the technique outlined above for the icons;
* Use ```<p>``` elements to represent the text messages.

For example:
```
<div class="pb-mod-fte-helperInfoArea">
<span `role="img" aria-label="Meeting details"` class="fte-ic30 fte-icon-info">Meeting information icon</span>
<span `role="img" aria-label="Help"` class="fte-ic30 fte-icon-help">Help icon</span>

<div class="fte-fig arraw1"></div>
<div class="fte-fig arraw2"></div>
<div class="fte-fig text1">`<p>`You can select the meeting information icon for meeting details.`</p>`</div>
<div class="fte-fig text2">`<p>`You can select the help icon for help.`</p>`</div>
</div>
```

Note: labelling needs to be consistent. Saying "Select the meeting information icon for meeting details" is confusing because the button itself is labelled "Meeting details". The above message should be changed to:
```
<div class="fte-fig text1">`<p>`You can select the meeting details icon for meeting information.`</p>`</div>
```
